docs/rules/g103_use_of_unsage_block.md

```diff
@@ -0,0 +1,53 @@
+---
+id: g103
+title: G103: Use of unsafe block
+---
+
+# G103: Use of unsafe block
+
+Using the unsafe package in Go gives you low-level memory management and many of the strength of the C language but also gives flexibility to the attacker of your application. The pointer arithmetic is one of the examples from the unsafe package which can be used for data leak, memory corruption or even execution of attackers own script.
+
+Also, you should keep in mind that the "unsafe" package is not protected by Go 1 compatibility guidelines.
```

Comment:
It would be helpful to have a link to `Go 1 compatibility guidelines`.

Reply:
You mean this:
https://golang.org/doc/go1compat#expectations

Reply:
I added it.